app/views/contexts/section_summary_context.py

```diff
@@ -117,10 +131,13 @@ def _build_summary(self, return_to: Optional[str]):
                 )
             }
 
-            return {**collapsible, **summary_elements}
+        else:
+            summary_elements = {}
 
         return {
+            **{"show_non_item_answers": show_non_item_answers},
```

Comment:
Does this need to be unpacked?
```suggestion
            "show_non_item_answers": show_non_item_answers,
```